src/test/groovy/org/prebid/server/functional/tests/BidValidationSpec.groovy

```diff
@@ -139,7 +139,7 @@ class BidValidationSpec extends BaseSpec {
         assert response.seatbid.size() == 0
 
         and: "PBS should emit an error"
-        assert response.ext?.warnings[ErrorType.PREBID]*.code == [999]
+        assert response.ext?.warnings[ErrorType.PREBID]*.code == [10008]
```

Comment:
I would propose to extract those codes to a class level constants or better to have one place where all warnings vars are stored (like in PrebidLog.java)